### PR TITLE
Updating version numbers

### PR DESCRIPTION
**Pull request by mfeigl**
### Description

Updating the version numbers in preparation for the 200.4 release

Checklist:
- [x] Set target branch to main (current release) or v.next (next release)
- [x] Request a reviewer
- [x] Assign a reviewer
- [x] Tag reviewer in comment

### CODE REVIEW

**Comment by mfeigl** [comment]
@mbcoder could you please review?

File: editing/edit-feature-attachments/build.gradle [hunk 1]
```diff
@@ -47,7 +47,7 @@ dependencies {
     natives "com.esri.arcgisruntime:arcgis-java-jnilibs:$arcgisVersion"
     natives "com.esri.arcgisruntime:arcgis-java-resources:$arcgisVersion"
     // handle SLF4J http://www.slf4j.org/codes.html#StaticLoggerBinder
-    implementation 'org.slf4j:slf4j-nop:2.0.9'
+    implementation 'org.slf4j:slf4j-nop:2.0.12'
     implementation 'commons-io:commons-io:2.15.0'
```
Comment (mbcoder) [inline editing/edit-feature-attachments/build.gradle]: Consider using https://mvnrepository.com/artifact/commons-io/commons-io/2.16.0

Needs testing however

**Review by mbcoder — CHANGES_REQUESTED** [review]
Spotted one lib we should consider updating, but looks good other than that

File: editing/edit-feature-attachments/build.gradle (hunk 1 above)
Comment (mfeigl) [reply editing/edit-feature-attachments/build.gradle]: Updated all 7 occurences and tested a couple which ran fine

**Review by mbcoder — APPROVED** [review]
Looks good

**Pull request merged**